packages/meteor-gazelle-createdby/lib/tests/createdByTester.js

```diff
@@ -0,0 +1,12 @@
+CreatedByTesters = new Mongo.Collection('createdByTesters');
+CreatedByTester = Astro.Class({
+  name: 'CreatedByTester',
+  collection: 'CreatedByTesters',
+  fields: {
+    postContent: 'string'
+  },
+  // Validators not should be added by the behavior.
```

Comment:
Im assuming you added this comment because of an error when including the package. @ajax-wcd  and myself are aware of this. You can go ahead and remove this comment - we will be working out the fix for the Validators package include soon.